elastic/kibana
# [AO] Clean up threshold action context variables

Closes #160275, closes #159715

## Summary

This PR cleans up the action variables. After a discussion with Chris and Katrin, we decided to remove metric, and threshold information from the action variables as we have them already in the rule, and at the time of defining the rule, the user can set the conditions manually in the message.

## 🧪 How to test
- Make sure `xpack.observability.unsafe.thresholdRule.enabled` is set to true in kibana yml config
- Create a threshold rule that fires with an action with all the `context.` prefix variables
- Check if the value is populated as expected
